### PR TITLE
devbox: add page

### PR DESCRIPTION
**Pull request by jeff-hykin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** devbox 1.x (only planned version, maybe in 5 years there will be a v2)

I think this is correct; I only added the md file to `common` since it is a mostly (e.g. probably not sunos) cross platform tool. Just let me know if I need to change anything

### CODE REVIEW

File: pages/common/devbox.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+devbox global install
+```
+
+- Install package for just this project:
+
+```
+devbox init
+devbox add {{package_name}}@{{version_number}}
+devbox shell
+```
```
Comment (acuteenvy) [inline pages/common/devbox.md:27]:
```suggestion
- Initialize a Devbox environment:

`devbox init`

- Search for packages:

`devbox search {{query}}`

- Add a package to `devbox.json` of the current environment:

`devbox add {{package}}@{{version}}`

- Add a package to the global `devbox.json`:

`devbox global add {{package}}@{{version}}`

- Install packages from `devbox.json` in the current environment:

`devbox install`

- Install packages from the global `devbox.json` system wide:

`devbox global install`

- Run a Devbox shell with installed packages:

`devbox shell`
```

**Review by acuteenvy — CHANGES_REQUESTED** [review]
Thanks for your contribution. The page has gotten multiple errors from CI, so please read the [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md). Multiline code blocks are not supported - every example should be a separate command. HTML tags are not supported. The last description line should be `More information: <https://example.org>`. I've fixed all the errors in these suggestions.

**Comment by jeff-hykin** [comment]
Thanks for the changes, I ran out of time when I was doing the original PR.

I think I'm going to close this PR because I feel that, for now, for this tool, the isolated commands are too misleading/problematic.

If we have to read the full description of three of the tldr commands, and manually figure out how to order them and which ones are required for which task (e.g. `init` isn't required for global install) it's really not a TLDR. Or rather I'd look at the tealdeer page and say "Cool, TLDR, now please tell me how do I actually install something???" and the answer requires two commands.

Having standards is perfectly fine, so I'm not going to ask for an exception, which means the alternative is to leave it out for now.

However, I'll talk to the `devbox` guys and see if I can get some of these commands to be combined. For example, having devbox auto-init when missing an init. If I can do that, then I can come back here, omit the init command, without making the tealdeer misleading or annoying.

**Comment by Magrid0** [comment]
> I think I'm going to close this PR because I feel that, for now, for this tool, the isolated commands are too misleading/problematic.

Is there any update or you just forgot to close it?

**Comment by jeff-hykin** [comment]
I forgot to close the PR, sorry about that.

**Pull request closed without merging**